components/ScrollToTopButton/ScrollToTopButton.module.scss

```diff
@@ -0,0 +1,23 @@
+.button {
```

Comment:
The way the button is placed is not really optimal. This is especially obvious on mobile:
<img width="501" alt="image" src="https://user-images.githubusercontent.com/1641218/168041849-800081af-7aa6-4190-88c9-e52f9462d6c1.png">

The sizing is also not good - the icon should be a bit bigger IMO, and the spacing is not even, which makes the icon appear not centered vertically.

Here's my take on the button placement:
- The clickable size should be a bit bigger, so that's it easy to click/tap (increasing icon size should help, but maybe it's worth to increase the padding up a notch
- The bottom and right offset should be smaller, at least on smaller screens
- On large screens, it would be nice to place the button closer to the content, e.g. place it in a virtual "container".